mlflow/server/__init__.py

```diff
@@ -246,6 +247,24 @@ def _build_gunicorn_command(gunicorn_opts, host, port, workers, app_name):
     ]
 
 
+def _build_uvicorn_command(uvicorn_opts, host, port, workers, app_name):
```

Comment:
Do we want to use the hybrid approach of running gunicorn's process management layer  (not WSGI) and have it delegate to uvicorn? 
IIRC, when we did the setup for AI Gateway's backend, we determined that we get a benefit in this setup where if a process dies, gunicorn can automatically restart an ASGI process. If just using base uvicorn, the server would need to be manually restarted to recover. 

`gunicorn -k uvicorn.workser.UvicornWorker -w workers app_name` is the process handling delegation mode.

Reply:
(in this config we still wouldn't accept gunicorn configs as we don't want to expose the WSGI interface)

Reply:
Thanks @BenWilson2  - awesome call-out! I've pushed this change :)

Reply:
Right - reverted